akka-projection-jdbc/src/main/scala/akka/projection/jdbc/internal/Dialect.scala

```diff
@@ -132,21 +140,25 @@ private[projection] case class MySQLDialect(schema: Option[String], tableName: S
   def this(tableName: String) = this(None, tableName)
 
   private val table = schema.map(s => s"$s.$tableName").getOrElse(tableName)
+
   override val createTableStatements =
     immutable.Seq(
       s"""
-     CREATE TABLE IF NOT EXISTS $table (
-      PROJECTION_NAME VARCHAR(255) NOT NULL,
-      PROJECTION_KEY VARCHAR(255) NOT NULL,
-      CURRENT_OFFSET VARCHAR(255) NOT NULL,
-      MANIFEST VARCHAR(4) NOT NULL,
-      MERGEABLE BOOLEAN NOT NULL,
-      LAST_UPDATED BIGINT NOT NULL,
-      PRIMARY KEY(PROJECTION_NAME,PROJECTION_KEY)
-     );
+      CREATE TABLE IF NOT EXISTS $table (
+        PROJECTION_NAME VARCHAR(255) NOT NULL,
+        PROJECTION_KEY VARCHAR(255) NOT NULL,
+        CURRENT_OFFSET VARCHAR(255) NOT NULL,
+        MANIFEST VARCHAR(4) NOT NULL,
+        MERGEABLE BOOLEAN NOT NULL,
+        LAST_UPDATED BIGINT NOT NULL,
+        PRIMARY KEY(PROJECTION_NAME,PROJECTION_KEY)
+       );
     """,
       // create index
-      s"""CREATE INDEX IF NOT EXISTS PROJECTION_NAME_INDEX ON $table (PROJECTION_NAME);""")
+      s"""CREATE INDEX PROJECTION_NAME_INDEX ON $table (PROJECTION_NAME);""")
```

Comment:
We are not yet fully telling the truth when we call the MySQL create table script from one of the methods named `createIfNotExists`. The index creation will fail. 

There are means to cover that in MySQL, but not so straightforward (see https://dba.stackexchange.com/questions/24531/mysql-create-index-if-not-exists). 

For the moment the way out is to call `dropIfExists` and then `createIfNotExists`. Which is pretty much ok for tests purposes.